### PR TITLE
Multi stage docs imprv

### PR DESCRIPTION
**Pull request by gortiz**
A slightly improvement in multi-stage descriptions, mainly changing the way stages are named and describing the stage types

### CODE REVIEW

File: reference/troubleshooting/troubleshoot-multi-stage-query-engine.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ CAST(timestampCol AS BITINT) >= longCol
 
 Default names for projections with function calls are different between v1 and v2.&#x20;
```
Comment (Jackie-Jiang) [inline reference/troubleshooting/troubleshoot-multi-stage-query-engine.md]: Not introduced in this PR, but should we call it single-stage and multi-stage engine?

File: reference/troubleshooting/troubleshoot-multi-stage-query-engine.md [hunk 2]
```diff
@@ -200,6 +201,14 @@ See [null handling support](null-value-support.md)
 
 * aggregate function that requires literal input (such as `percentile`, `firstWithTime`) might result in a non-compilable query plan when used in v2.&#x20;
 
+### Byte literals
+
+<!-- TODO: Explain how to use byte literals -->
```
Comment (Jackie-Jiang) [inline reference/troubleshooting/troubleshoot-multi-stage-query-engine.md]: Are you planning to fill this part in this PR or a separate PR?

Comment (gortiz) [reply reference/troubleshooting/troubleshoot-multi-stage-query-engine.md]: Yes, this two TODOs shouldn't be here. ~I'm going to remove them~

Comment (gortiz) [reply reference/troubleshooting/troubleshoot-multi-stage-query-engine.md]: I've decided to just add a short explanation for these two topics

File: reference/troubleshooting/troubleshoot-multi-stage-query-engine.md (hunk 1 above)
Comment (gortiz) [reply reference/troubleshooting/troubleshoot-multi-stage-query-engine.md]: Yes. There are tons of mentions to v1 and v2. I've pushed a commit changing most of them. If you prefer I can cherry pick that commit into another PR

**Pull request merged**